bootstrap/src/components/Header/Links.js

```diff
@@ -13,9 +16,11 @@ const Links = () => {
             >
                 Welcome
             </NavLink>
-            <NavLink to="/" className={styles.Link} activeClassName={styles.LinkActive} exact>
-                Home
-            </NavLink>
+            {projectId && (
```

Comment:
Why do we need projcetId for the Home link?

Reply:
As it is assumed that components in Home page always depend on projectId. But it's probably overengineered. Let's leave it up to user. I will remove it.